strawberryfields/backends/tfbackend/backend.py

```diff
@@ -362,12 +227,19 @@ def state(self, modes=None, **kwargs):
         r"""Returns the state of the quantum simulation, restricted to the subsystems defined by `modes`.
 
         Args:
-            modes (int or Sequence[int]): specifies the mode or modes to restrict the return state to.
-                                          This argument is optional; the default value ``modes=None`` returns the state containing all modes.
-            **kwargs: optional keyword args (`session`: a Tensorflow session; `feed_dict`: a Python dictionary feeding the desired numerical values for Tensors)
-                      which will be used by the Tensorflow simulator for numerically evaluating the measurement results.
+            modes (int or Sequence[int] or None): Specifies the modes to restrict the return state to.
+                None returns the state containing all the modes.
+                If modes is not ordered, the returned state contains the requested modes in the given order, i.e.,
```

Comment:
If modes **is** ordered?

```suggestion
                If ``modes`` is ordered, the returned state contains the requested modes in the given order, i.e.,
```